stdlib/public/Distributed/CMakeLists.txt

```diff
@@ -45,10 +46,15 @@ add_swift_target_library(swiftDistributed ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS
   C_COMPILE_FLAGS
     -DswiftDistributed_EXPORTS
     -I${SWIFT_SOURCE_DIR}/stdlib/include
+
   SWIFT_COMPILE_FLAGS
     ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS}
     -parse-stdlib
-  LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
+    "-enable-experimental-feature" "Macros"
+    "-enable-experimental-feature" "ExtensionMacros"
```

Comment:
We shouldn't need either of these experimental features; they've been enabled by default for quite a while.

Reply:
Gotcha cleaning up then